bindings/java/iota-wallet-java/examples/src/GenerateAddress.java

```diff
@@ -15,18 +15,23 @@ public static void main(String[] args) throws WalletException, InitializeWalletE
         // This example assumes that a wallet has already been created using the ´CreateWallet.java´ example.
         // If you have not run the ´CreateAccount.java´ example yet, run it first to ensure that the wallet can be loaded correctly.
         Wallet wallet = new Wallet(new WalletConfig()
-                .withClientOptions(new ClientConfig().withNodes("https://api.testnet.shimmer.network"))
-                .withSecretManager(new StrongholdSecretManager("PASSWORD_FOR_ENCRYPTION", null, "example-wallet"))
+                .withClientOptions(new ClientConfig().withNodes(Env.NODE))
+                .withSecretManager(
+                        new StrongholdSecretManager(Env.STRONGHOLD_PASSWORD, null, Env.STRONGHOLD_SNAPSHOT_PATH))
                 .withCoinType(CoinType.Shimmer)
         );
 
         // Get account and sync it with the registered node to ensure that its balances are up-to-date.
-        AccountHandle a = wallet.getAccount(new AccountAlias("Alice"));
+        AccountHandle a = wallet.getAccount(new AccountAlias(Env.ACCOUNT_NAME));
         a.syncAccount(new SyncAccount().withOptions(new SyncOptions()));
 
         // Generate two addresses.
         AccountAddress[] addresses = a.generateAddresses(new GenerateAddresses().withAmount(2));
 
+        // Print the generated addresses.
+        for (AccountAddress address : addresses)
+            System.out.println(address.getAddress());
+
         // Print all addresses.
         for (AccountAddress address : a.getPublicAddresses())
             System.out.println(address.getAddress());
```

Comment:
```suggestion
```

lets remove that, we are already printing the generated addresses above (which this example is about)